src/components/ApplicationViews.js

```diff
@@ -1,13 +1,22 @@
 import React from "react"
 import { Route } from "react-router-dom"
+import { ArticleProvider } from "./article/ArticleProvider";
+import { ArticleList } from "./article/ArticleList";
+import { ArticleDetail } from "./article/ArticleDetail";
 
 export const ApplicationViews = () => {
   return (
     <>
-
+      <ArticleProvider>
       <Route exact path="/">
```

Comment:
I think after you delete an article it takes you back to /articles which doesnt show the article list there